### PR TITLE
Prepare extended lab

### PR DESCRIPTION
**Pull request by gs1571**

### CODE REVIEW

File: labs/topologies/lab10_dc_cisco/src/lab_generators/bgp.py [hunk 1]
```diff
@@ -42,33 +44,42 @@ def run_cisco(self, device: Device):
 
         if not asnum or not rid:
             return
-        with self.block("router bgp", asnum):
+        with (self.block("router bgp", asnum)):
```
Comment (gregory-mac) [inline labs/topologies/lab10_dc_cisco/src/lab_generators/bgp.py]: Redundant parentheses?

File: labs/topologies/lab11_dc_frr/src/lab_generators/bgp.py [hunk 2]
```diff
@@ -42,33 +44,42 @@ def run_cisco(self, device: Device):
 
         if not asnum or not rid:
             return
-        with self.block("router bgp", asnum):
+        with (self.block("router bgp", asnum)):
```
Comment (gregory-mac) [inline labs/topologies/lab11_dc_frr/src/lab_generators/bgp.py]: Same.

File: labs/topologies/lab12_dc_arista_cisco_frr/src/lab_generators/bgp.py [hunk 3]
```diff
@@ -42,33 +44,42 @@ def run_cisco(self, device: Device):
 
         if not asnum or not rid:
             return
-        with self.block("router bgp", asnum):
+        with (self.block("router bgp", asnum)):
```
Comment (gregory-mac) [inline labs/topologies/lab12_dc_arista_cisco_frr/src/lab_generators/bgp.py]: Same.

**Review by gregory-mac — APPROVED** [review]

**Comment by vadvolo** [comment]
ok

File: labs/topologies/lab10_dc_cisco/src/lab_generators/bgp.py (hunk 1 above)
Comment (gs1571) [reply labs/topologies/lab10_dc_cisco/src/lab_generators/bgp.py]: fix

**Review by gescheit — APPROVED** [review]

**Pull request merged**